share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -317,6 +317,37 @@ endmacro()
 macro(rocm_set_comp_cpackvar HEADER_ONLY components)
     # Setting component specific variables
     set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+    if(NOT CPACK_RESOURCE_FILE_LICENSE)
+        file(GLOB _license_files LIST_DIRECTORIES FALSE "${CMAKE_SOURCE_DIR}/LICENSE*")
```

Comment:
There are 3 different license files names that we use. Should we limit it to just those the files? This may address @cgmb 's concerns

Reply:
Seems like a reasonable solution to me, @cgmb? I will also note that it only looks for LICENSE files in the top-level CMake directory, which should help to ensure that only license files for the intended library are used.

Reply:
I was mostly worried about the use of `${CMAKE_SOURCE_DIR}`, but given that it's not recursive and you've added a check to ensure we find something, that's probably sufficient.

Reply:
That is, I think it's fine as is.